Marlin/src/feature/runout.cpp

```diff
@@ -84,15 +84,11 @@ void event_filament_runout() {
     }
   #endif
 
-  TERN_(EXTENSIBLE_UI, ExtUI::onFilamentRunout(ExtUI::getActiveTool()));
-
-  #if EITHER(HOST_PROMPT_SUPPORT, HOST_ACTION_COMMANDS)
-    const char tool = '0'
-      #if NUM_RUNOUT_SENSORS > 1
-        + active_extruder
-      #endif
-    ;
-  #endif
+  const char tool = '0'
+    #if E_STEPPERS > 1
+      + extruder
+    #endif
+  ;
```

Comment:
You just dropped the ExtUI code?

Reply:
True, that was a mistake. It’s fixed in the last commit.